flocker/control/_model.py

```diff
@@ -199,6 +201,7 @@ class Application(PRecord):
     restart_policy = field(mandatory=True, initial=RestartNever())
     environment = field(mandatory=True, initial=pmap(), factory=pmap,
                         type=PMap)
+    running = field(mandatory=True, initial=True, type=bool)
 
```

Comment:
:question: Deployer discovery will determine the actual running state of an application, but should we maybe assume that an `Application` is not running until a host confirms otherwise?